playbooks/create_hosted_cluster.yaml

```
@@ -4,17 +4,77 @@
   hosts: kvm_host_hypershift
   vars_files:
     - "{{playbook_dir}}/secrets.yaml"
+  tasks:
+    - name: Setting host
+      set_fact:
+        host: 'kvm_host_hypershift'
+    - name: Install Prereqs on host
+      import_role:
+        name:  install_prerequisites_host_hypershift
 
-  roles:
-  - install_prerequisites_host_hypershift
 
 
+- name: Create macvtap network
+  hosts: kvm_host_hypershift
+  tasks:
+    - name: Setting interface name
+      set_fact:
+        networking:
+          device1: "{{ hypershift.networking_device }}"
+    - name:  Creating macvtap network
+      import_role:
+        name: macvtap
 
-- name: Create AgentServiceConfig Hosted Control Plane and InfraEnv
+
+
+- name: Create bastion for hypershift
   hosts: kvm_host_hypershift
+  vars_files:
+    - "{{playbook_dir}}/secrets.yaml"
+  tasks:
+    - name: Creating Bastion
+      include_role:
+        name: create_bastion_hypershift
+      when: hypershift.create_bastion == 'true'
+
+
+- name: Configuring Bastion
+  hosts: bastion_hypershift
+  become: true
+  vars_files:
+    - "{{playbook_dir}}/secrets.yaml"
+  tasks:
+    - name: Setting host
+      set_fact:
+        host: 'bastion_hypershift'
+    - name: Install Prereqs
+      import_role:
+        name: install_prerequisites_host_hypershift
+    - name: Configure Bastion
```

```suggestion
        host: 'bastion_hypershift'
        
    - name: Install Prereqs
      import_role:
        name: install_prerequisites_host_hypershift
        
    - name: Configure Bastion
```
Add single space between tasks